docusaurus/docs/reference/ui-extensions/addComponent.md

```diff
@@ -0,0 +1,51 @@
+---
+id: add-component
+title: addComponent()
```

Comment:
not sure what folks think about removing all () from the titles? Probably not needed in the sidebar

Reply:
can we have that level of nesting in the menu though?

Reply:
Are you still suggesting that addComponent (for example) would have its own page, if so does the menu actually allow that level of nesting?

Reply:
I like what @sunker is suggesting!

Reply:
> you still suggesting that addComponent (for example) would have its own page, if so does the menu actually allow that level of nesting?

No everything on the same page. The example above describes the structure for all headings in the page `developers/plugin-tools/reference/ui-extensions`.

Reply:
No strong opinions here - will defer to your judgement then :)

Reply:
Took a stab at changing this. Please have a look in the [dev env](https://grafana-dev.com/developers/plugin-tools/reference/ui-extensions-2) and let me know what you think. (kept the old structure so you can compare)